Doc/library/functools.rst.bak

```diff
@@ -0,0 +1,511 @@
+:mod:`functools` --- Higher-order functions and operations on callable objects
```

Comment:
This shouldn't have been included in the commit.

Reply:
Thanks. I don't know why this isn't ignored by git. I will remove it.